src/common/enums/logical_operator_type.cpp

```diff
@@ -35,6 +35,8 @@ string LogicalOperatorToString(LogicalOperatorType type) {
 		return "LIMIT";
 	case LogicalOperatorType::ORDER_BY:
 		return "ORDER_BY";
+	case LogicalOperatorType::ORDER_BY_LIMIT:
```

Comment:
Could you change this to "TOP_N" instead of "ORDER_BY_LIMIT"? Only a stylistic change but I think it conveys the meaning more clearly.

Reply:
Agree. I will make this change.